dataprofiler/data_readers/avro_data.py

```diff
@@ -11,9 +13,9 @@
 class AVROData(JSONData, BaseData):
     """AVROData class to save and load spreadsheet data."""
 
-    data_type = "avro"
+    data_type: Optional[str] = "avro"
```

Comment:
wouldn't this just be `str` not `Optional[str]` ... not sure we would want to allow for `None` https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type

Reply:
@taylorfturner I wonder if it has to do with the Abstract being `None` until defined. But agreed, if it can be str to be str

Reply:
When I annotate with `str`, mypy complains because it's set to `None` in `BaseData`.